### DISCUSSION TITLE
Jogos Digitais

### DISCUSSION THREAD

**Discussion by rodrigets** [frontendbr/forum#1342] [Discussão]
Olá pessoal, tudo bem?
Atualmente sou desenvolvedor full stack e tenho vontade de fazer o curso tecnólogo de jogos digitais.
Gostaria de saber quais experiencias tiveram nos estudos e no mercado de trabalho e quais faculdades recomendam.
Valeu!

**Comment by gabriel-hahn** [comment]
Olá Rodrigo.

Não sei o nível que tu se encontra no desenvolvimento de jogos, se pensa em desenvolvimento utilizando uma Unity ou Unreal e etc. Na parte de ensino online, já curti bastaste os vídeos do **Mark Price** da **Devslopes**, assim como alguns outros cursos na Udemy, ensinando a fazer alguns jogos na prática.
Estes tipos de curso não vão te dar um baita diploma, mas já mostra um pouco do mundo de desenvolvimento de games e te orienta na hora de buscar uma especialização no assunto.

É uma boa forma de começar, caso tu ainda seja iniciante :)

**Comment by rodrigets** [comment]
@gabriel-hahn , Obrigado pelas dicas.
Eu fiz apenas dois jogos, o primeiro fiz utilizando Pygames e o outro Godot Engine.
São jogos simples, mas me fizeram pensar em estudar mais esta area e desenvolver jogos mais complexos.
Vou fazer um curso de Unity na Udemy para ver como funciona.

**Comment by Matheus-de-Souza** [comment]
Olá @gabriel-hahn, 

eu sou formado em [Tecnologia em Jogos Digitais](http://unisinos.br/vestibular/curso/jogos-digitais/sao-leopoldo) pela universidade Unisinos/RS. Recomendo muito o curso na Unisinos para quem quer trabalhar na área, e perceber que desenvolver jogos "não é brincadeira", hehe. Eu continuo trabalhando com software hoje, mas saí da área de jogos porque percebi que não gostava tanto da área quanto imaginava ao entrar no curso, hehe. Mas tenho amigos do curso que se esforçaram bastante e trabalham na Rockstar, por exemplo, e outras empresas conhecidas do ramo, alguns formaram empresas para desenvolver seus próprios jogos também. 

Meu conselho mesmo, é que você tenha clareza quanto ao esforço de trabalhar nessa área e se você tem vontade mesmo, vai enfrente os desafios e faça bons games. 😄 

PS.: A Unity é a melhor engine para fazer jogos Indie hoje em dia. Na [Asset Store](https://assetstore.unity.com/) você encontra milhares de assets para fazer seus jogos, e muitos de graças.
PPS.: Tem um outro curso de [Tecnologia em Jogos Digitais](https://www.feevale.br/graduacao/jogos-digitais) na universidade Feevale Novo Hamburgo/RS, mas esse é mais focado nos artistas 3D do que nos programadores.

**Comment by allysonsouza** [comment]
Vale a pena também dar uma estudada nessa engine open-source que está ganhando espaço, a [Godot](https://godotengine.org/)

**Comment by rodrigets** [comment]
@allysonsouza , eu fiz alguns jogos 2D com a Godot.
Achei tranquilo, o gdscript é muito parecido com python.
